packages/flutter/test/scheduler/animation_test.dart

```diff
@@ -13,7 +13,11 @@ import 'scheduler_tester.dart';
 class TestSchedulerBinding extends BindingBase with ServicesBinding, SchedulerBinding { }
 
 void main() {
-  final SchedulerBinding scheduler = TestSchedulerBinding();
+  SchedulerBinding scheduler;
+
+  setUpAll(() {
+    scheduler = TestSchedulerBinding();
+  });
```

Comment:
this should be semantically identical. that it isn't is concerning. can you elaborate?

Reply:
Yeah, this looks like a bug in the web engine.

Reply:
I moved the test initialization logic into the generated test entrypoints and this issue went away 🎊